actions/createTokenizedRealm.ts

```diff
@@ -43,31 +62,86 @@ export default async function createTokenizedRealm({
       []
     )
     console.log('CREATE GOV TOKEN REALM: sending transactions')
+    console.log("Current instructions", {
+        realmInstructions,
+        mintsSetupInstructions,
+        councilMembersInstructions,
+    });
+    console.log("plugins", pluginList);
+
+    const pluginSigners: Keypair[] = []
+    const pluginIxes: TransactionInstruction[] = []
+
+    let predecessorProgramId: PublicKey | undefined = undefined;
+
+
+    if (pluginList.includes('gateway')) {
+      // By default, use Civic's uniqueness pass. TODO allow this to be overridden in advanced mode.
+      const passType = new PublicKey(defaultSybilResistancePass.value)
```

Comment:
note @chriteixeira this is where you would pass in a value set from the form in advanced mode